anta/custom_types.py

```diff
@@ -138,6 +138,12 @@ def validate_regex(value: str) -> str:
     BeforeValidator(interface_autocomplete),
     BeforeValidator(interface_case_sensitivity),
 ]
+PortChannelInterface = Annotated[
+    str,
+    Field(pattern=r"^Port-Channel[0-9]{1,3}$"),
```

Comment:
Can you move the regex to a constant and add unit test please?

Reply:
Added constant and unit test for the same. Thanks!!